Back End Challenge/pom.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.test.headstorm</groupId>
+  <artifactId>challenge</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>war</packaging>
+
+  <name>challenge Maven Webapp</name>
+  <!-- FIXME change it to the project's website -->
+  <url>http://www.example.com</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+
+    <spring.version>5.0.0.RELEASE</spring.version>
+    <spring.webmvc.version>5.0.0.RELEASE</spring.webmvc.version>
+    <slf4j.version>1.7.5</slf4j.version>
+    <logback.version>1.0.13</logback.version>
+    <logback.extensions.version>0.1.1</logback.extensions.version>
+
+    <mybatis.version>3.2.2</mybatis.version>
+    <oracle.version>11.2.0.3</oracle.version>
+
+    <junit.version>4.11</junit.version>
+    <jackson.version>1.9.13</jackson.version>
+
+    <gson.version>2.8.0</gson.version>
+  </properties>
+
+  <dependencies>
+<!--    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>servlet-api</artifactId>
+      <version>2.5</version>
+    </dependency>-->
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>${junit.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>servlet-api</artifactId>
+      <version>2.5</version>
+    </dependency>
+
+    <!-- framework -->
+    <dependency>
+      <groupId>org.springframework</groupId>

```

Comment:
How did you decide on using the Spring Framework to solve this challenge? 
What are the advantages, and very importantly, what are the disadvantages to using the Spring Framework in relation to other choices?